pkg/provision/kind.go

```diff
@@ -133,34 +137,24 @@ func KindCluster(platform *platform.Platform) error {
 	return platform.GetKubectl()("delete sc standard")
 }
 
+// createKubeAdmPatches reads a Platform config, creates a new ClusterConfiguration from it and
+// then extracts a slice of kind-specific KubeAdm patches from it.
 func createKubeAdmPatches(platform *platform.Platform) ([]string, error) {
 	clusterConfig := kubeadm.NewClusterConfig(platform)
 	clusterConfig.ControlPlaneEndpoint = ""
-	clusterConfig.ClusterName = ""
+	clusterConfig.ClusterName = platform.Name
 	clusterConfig.APIServer.CertSANs = nil
-	clusterConfig.APIServer.ExtraVolumes = []api.HostPathMount{
-		{
-			Name:      "oidc-certificates",
-			HostPath:  path.Join(kindCADir, filepath.Base(platform.IngressCA.Cert)),
-			MountPath: "/etc/ssl/oidc/ingress-ca.pem",
-			ReadOnly:  true,
-			PathType:  api.HostPathFile,
-		},
-	}
-	clusterConfig.APIServer.ExtraArgs["oidc-ca-file"] = "/etc/ssl/oidc/ingress-ca.pem"
 
-	if platform.Kubernetes.AuditConfig.PolicyFile != "" {
-		clusterConfig.APIServer.ExtraArgs["audit-policy-file"] = "/etc/kubernetes/policies/audit-policy.yaml"
+	vol := &clusterConfig.APIServer.ExtraVolumes
```

Comment:
since the audit configs mappings are set upstream in `NewClusterConfig` this needs to be amended to _append_ rather than _set_ the `oidc-certificate` mapping.